subjects/unzipstring/README.md

```diff
@@ -0,0 +1,54 @@
+## Unzipstring
+
+### Instructions
+
+write a function named `Unzipstring` that takes a string in form of a number and an alphabet (like 3w or 2m3e) then returns a new string (3w ==> www and 2m3e ==> mmeee).
```

Comment:
Try this...

Write a function called `Unzipstring` that takes a string that will be a kind of code and your function will have to decrypt it and return a new string with the output.

It works as follows:

The string will be formed by a number followed by a letter, and the purpose is to print this letter the number of times that is requested by the number that precedes it.
```
"3w" ==> www
"2m3e" ==> mmee
```
- The number before the letter must be between 0 and 9.
- The Letter after each number must be between `a and z` or `A and Z`.
- You cannot have two numbers or two letters in a row.
- If the input string does not respect the format, return `Invalid Input`.